### ISSUE TITLE
Postman begins timeouting after a period of inactivity 

### ISSUE THREAD

**Issue by BladeMF** [a85/POSTMan-Chrome-Extension#13072]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

After a period of inactivity Postman desktop beginst timing out _all_ requests with this in the console:
```
GET http://google.com/
Error: connect ETIMEDOUT 2a00:1450:4017:813::200e:80
Request Headers
User-Agent: PostmanRuntime/7.41.0
Accept: */*
Cache-Control: no-cache
Postman-Token: 1227481e-a750-406f-9077-c078e3e92c9d
Host: google.com
Accept-Encoding: gzip, deflate, br
Connection: keep-alive
```
The computer still has internet and using the web version does produce the same results, i.e. it works as expected. The only solution I've come up with apart from restarting computer is disable and then re-enable the network card.

### Steps To Reproduce

It is not really reproducible, but it happens _multiple times_ per day, _every_ day. Tried 

### Screenshots or Videos

![image](https://github.com/user-attachments/assets/b8a915f0-9b8d-4d69-b2fb-71bb9b218332)

### Operating System

Windows

### Postman Version

11.7.0

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by BladeMF** [comment]
Just updated to 11.8 while being stuck on a timeout and the requests work. In other cases just restarting Postman does not solve the problem.

**Comment by codenirvana** [comment]
@BladeMF do you have a proxy or VPN setup by any chance?

**Comment by BladeMF** [comment]
No. I am using mobile internet though, but as far as my computer is concerned it's just wifi.
